src/main/java/com/google/sps/servlets/DeletePlaceGuideServlet.java

```diff
@@ -20,8 +20,9 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String id = request.getParameter("id");
+    String currentUrl = request.getParameter("currentUrl");
```

Comment:
You should modify the PlaceGuideRepository javascript class so that it sends the currentUrl parameter to the servlet.

Reply:
added to query params.